File: MODUL_5/C/PRAK504_2310817310014_MUHAMMADRIZKISAPUTRA.c
#include <stdio.h>

int reverse(int number) {
    int reversedNumber = 0;

    while (number != 0) {
        reversedNumber = (reversedNumber * 10) + (number % 10);
        number /= 10;
    }

    return reversedNumber;
}

int main() {
    int A, B;
    scanf("%d %d", &A, &B);

    A = reverse(A);
    B = reverse(B);

    int C = A + B;
    printf("%d", reverse(C));

    return 0;
}

File: MODUL_5/Py/PRAK504_2310817310014_MUHAMMADRIZKISAPUTRA.py
def reverse_number(number):
    reversed_number = 0

    while number != 0:
        reversed_number = (reversed_number) * 10 + (number % 10)
        number //= 10

    return reversed_number

A, B = map(int, input("Masukkan bilangan : ").split(" "))

reversed_A = reverse_number(A)
reversed_B = reverse_number(B)

result = reversed_A + reversed_B

print(f"Hasilnya adalah : {reverse_number(result)}")
